eqs/src/eth_polling.rs

```diff
@@ -218,6 +220,29 @@ impl EthPolling {
                                 merkle_tree.frontier();
                         }
 
+                        //update transaction_by_id and transaction_id_by_hash hashmap
+                        let mut record_index = 0;
+                        transitions
+                            .iter()
+                            .enumerate()
+                            .for_each(|(txn_id, transition)| {
+                                updated_state.transaction_by_id.insert(
+                                    (meta.block_number.as_u64(), txn_id as u64),
+                                    cap_rust_sandbox::ledger::CommittedCapeTransition {
+                                        block_id: meta.block_number.as_u64(),
+                                        txn_id: txn_id as u64,
+                                        output_start: uids[record_index],
```

Comment:
It seems like uids has a similar redundancy, but that's something we can clean up later.